lib/validate.js

```diff
@@ -1,76 +1,74 @@
-var merge = require( "mout/object/merge" ),
-	semver = require( "semver" ),
-	subjectExceptions = /^(fixup|squash)!|^\[[^\]]+\]:/,
-	defaults = {
-		component: true,
-		components: [],
-		limits: {
-			subject: 72,
-			other: 80
-		}
-	};
+var merge = require('mout/object/merge'),
```

Comment:
Let's use this opportunity to get rid of onevar as well. Make each of these use their own `var`